content/2021/2021-02/2021-02-15-pulsar-weekly.md

```diff
@@ -0,0 +1,187 @@
+
+- [Build] Reduce the `pulsar-test-latest-version` docker image size.
+
+    https://github.com/apache/pulsar/pull/9627 (@[lhotari](https://github.com/lhotari))
+
+- [Flaky Test] Fix the Bookie's `dbStorage_*CacheMaxSizeMb` setting in `pulsar-test-latest-version` docker image.
+
+    https://github.com/apache/pulsar/pull/9623 (@[lhotari](https://github.com/lhotari))
+
+- [Broker] Fix the timeout operation when the `wait-for-exclusive` producer is handled incorrectly.
+
+    https://github.com/apache/pulsar/pull/9600 (@[merlimat](https://github.com/merlimat))
+
+- [Client] Add `BouncyCastleProvider` as the security provider to prevent NPE.
+
+    https://github.com/apache/pulsar/pull/9601 (@[massakam](https://github.com/massakam))
+
+- [Build] Fix expired TLS certificates for C++ tests.
+
+    https://github.com/apache/pulsar/pull/9607 (@[sijie](https://github.com/sijie))
+
+- [Broker] Fix NPE that occurs in `PersistentStickyKeyDispatcherMultipleConsumers` when debug log is enabled.
+
+    https://github.com/apache/pulsar/pull/9587 (@[massakam](https://github.com/massakam))
+
+- [Broker] Only log `auth _errors_ ` at the error level.
+
+    https://github.com/apache/pulsar/pull/9325 (@[ivankelly](https://github.com/ivankelly))
+
+- [Admin] Fix the NPE that occurs when getting messages by the ID while the message is null.
+
+    https://github.com/apache/pulsar/pull/9537 (@[315157973](https://github.com/315157973))
+
+- [Admin CLI] Inform users when the request for expiring messages is not executed.
+
+    https://github.com/apache/pulsar/pull/9561 (@[MarvinCai](https://github.com/MarvinCai))
+
+- [Build] Disable looking up snapshot versions in public repositories.
+
+    https://github.com/apache/pulsar/pull/9336 (@[lhotari](https://github.com/lhotari))
+
+- [Python] Match BookKeeper's `grpcio` requirement.
+
+    https://github.com/apache/pulsar/pull/9569 (@[jbellis](https://github.com/jbellis))
+
+- [Broker] Fix the race condition in `BrokerService` topic cache.
+
+    https://github.com/apache/pulsar/pull/9565 (@[lhotari](https://github.com/lhotari))
+
+- [Client] Clean up consumers on multi-topic subscribe failure.
+
+    https://github.com/apache/pulsar/pull/9419 (@[addisonj](https://github.com/addisonj))
+
+- [FlakyTest] Fix NPEs in test cleanup methods.
+
+    https://github.com/apache/pulsar/pull/9442 (@[Wzhipeng](https://github.com/Wzhipeng))
+
+- [FlakyTest] Fix failed `LoadReportNetworkLimit` test.
+
+    https://github.com/apache/pulsar/pull/9581 (@[lhotari](https://github.com/lhotari))
+
+- [Build] Avoid introducing BookKeeper-common into the pulsar-common.
+
+    https://github.com/apache/pulsar/pull/9551 (@[zymap](https://github.com/zymap))
+
+### Event / News
+
+- [TGIP] Weekly live stream about Pulsar and its ecosystem.
+
+    - 019: Latest Updates On Apache Pulsar
+  
+        - https://www.youtube.com/watch?v=EkRULmkaWGY
+
+### Blog / Article
+
+- StreamNative's 2020 Year in Review
```

Comment:
since this is from 2/8-2/14, and this blog is published on 2/16, shall we move it to another file?